README.md

```diff
@@ -69,3 +102,9 @@ TODO:
 Create script to create wsk actions, populate credentials using .env files
 
 Port all issues from [Taiga kanban](https://tree.taiga.io/project/kalonb91-lang/) to this repository
+
```

Comment:
Should there be a section for "Running the application", like what to type in the chatroom?

Or a section on "Troubleshooting" to address common pitfalls?